packages/runtime/src/parser/parser.ts

```diff
@@ -960,3 +960,30 @@ export function getProgramVersion(program: string): number {
 	opcodeFromSentence(wordsFromLine(firstLine), 1, ip, ExecutionMode.APPLICATION);
 	return ip.tealVersion;
 }
+
+/**
+ *
+ * @param str string to convert
+ * @returns camelCase to hypenated string eg(fooBar to foo-bar)
+ */
+export function convertCapitalToHyphens(str: string): string {
+	return str.replace(/[A-Z]/g, (m) => "-" + m.toLowerCase());
+}
+
+/**
+ *
+ * @param object
+ * @returns object keys parsed from camelCase to hypenated str eg(fooBar to foo-bar)
```

Comment:
both functions have the same examples (fooBar to foo-bar). Is this correct?

Reply:
yeah since one function uses the other one to return the value, it's just that one decode string and other one object